WebPhapp/backend/spec.js

```diff
@@ -125,19 +125,84 @@ describe("loading express", function() {
   });
 
   it("test-route-patients-last", function(done) {
-    // test for fully qualified first and last name
+    // test for fully qualified last name
     request(server)
       .get("/api/v1/patients?last=krantz")
       .expect(function(res) {
         if(res.body.length <= 1) throw new Error('Should be two or more krantz patients for testing.');
-        if([res.body[0].first, res.body[0].last, res.body[0].dob, res.body[0].patient_id].includes(undefined)){
+        if([res.body[0].first, res.body[0].last, res.body[0].dob, res.body[0].patientID].includes(undefined)){
           throw new Error('Found empty patient field.');
         }
         return true;
       })
       .end(done);
   });
 
+  it("test-route-patients-null", function(done) {
```

Comment:
Why is the testing file called spec.js?

Reply:
Great looking test though!

Reply:
Spec for specification. It is common notation for JS testing. To me it makes sense that a unit test would help enforce a specification. Thanks!